runtime/missingmember_test.go

```diff
@@ -3542,18 +3542,18 @@ access(all) contract AuctionDutch {
 		access(all) let currentPrice: UFix64
 		access(all) let totalItems: Int
 		access(all) let acceptedBids: Int
-		access(all) let tickStatus: {UFix64:TickStatus}
-		access(all) let metadata: {String:String}
-
-		init(status:String, currentPrice: UFix64, totalItems: Int, acceptedBids:Int,  startTime: UFix64, tickStatus: {UFix64:TickStatus}, metadata: {String:String}){
-			self.status=status
-			self.currentPrice=currentPrice
-			self.totalItems=totalItems
-			self.acceptedBids=acceptedBids
-			self.startTime=startTime
-			self.currentTime= 42.0 // Clock.time()
-			self.tickStatus=tickStatus
-			self.metadata=metadata
+		access(all) let tickStatus: &{UFix64:TickStatus}
+		access(all) let metadata: &{String:String}
```

Comment:
Why doe the field types have to be changed to references?

Reply:
These `tickStatus` and `metadata` fields are populated from fields of a reference. So either we need to change the type of these two fields, or have to change the entire test to work with owned values. The former is the minimal change so went with that.